src/Webfactory/Slimdump/Config/Column.php

```diff
@@ -62,6 +62,14 @@ public function processRowValue($value) {
         }
 
         if ($this->dump == Config::REPLACE) {
```

Comment:
Nothing for this PR to care about, but I smell an emerging switch statement, which could be replaced in a more object oriented way. I haven't thought about it yet, but some kind of configurable Provider comes into mind. Depending on the solution, one might even get rid of the hard Faker dependency.

But again, this is more of a reminder for ourselves than a critique on the PR, which is fine with me. Thanks!